pkg/sources/git/git.go

```diff
@@ -329,7 +374,21 @@ func CloneRepo(ctx context.Context, userInfo *url.Userinfo, gitURL string, args
 	if err = GitCmdCheck(); err != nil {
 		return "", nil, err
 	}
-	clonePath, err := os.MkdirTemp(os.TempDir(), "trufflehog")
+
+	pids, _ := getScannerPIDs(ctx)
+
+	for _, pid := range pids {
+		err := cleanTempDir(ctx, pid)
```

Comment:
isn't this going to delete all the temp dirs for active trufflehog processes? isn't that the opposite of what we want?

Reply:
I might just leave a comment to explain it; it will only delete dirs that do *not* match those PID values.